pkgs/applications/version-management/jiri/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, lib
+, fetchurl
+, fetchzip
+, unzip
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation {
+  pname = "jiri";
+  version = "unstable-2022-07-26";
+
+  src =
+    let
+      base = "https://chrome-infra-packages.appspot.com/dl/fuchsia/tools/jiri";
+      x86_64 = "amd64";
```

Comment:
I don't get why we do this.

Reply:
This was addressing the previous comment on moving the block into a `let in` that is in the mkDerivation directly and is not used in a giant fallback input.